AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -26,6 +26,9 @@ adomainscan.com#$##wpsafe-wait1 { display: none !important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/81073
 azpk.in#$##main { display: block !important; }
 azpk.in#$##mdtimer_clocks { display: none !important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/86506
+siyahfilmizle.com#$##wrapper { padding-top: 0px !important; }
+siyahfilmizle.com#$##wrapper { margin-top: 0px !important; }
```

Comment:
Just `siyahfilmizle.com#$##wrapper { padding-top: 0 !important; }`. Why do you need the second rule?
Also you added rules to the wrong filter instead of Turkish filter.

Reply:
@zloyden, mobile layout has margin instead of padding.

Reply:
Then add the second rule to the mobile filter.